app/servises/article_analyzer.rb

```diff
@@ -1,7 +1,14 @@
 require 'nokogiri'
 require 'open-uri'
+require 'net/https'
+require 'uri'
+require 'json'
 
 class ArticleAnalyzer
+
+  ENDPOINT = "https://article-analazer-anendo8.cognitiveservices.azure.com/"
```

Comment:
make the strings consistent, use single quotes. also use freeze to make them immutable